docs/samples/v1beta1/transformer/torchserve_image_transformer/local_config.properties

```diff
@@ -0,0 +1 @@
+model_snapshot={"name":"startup.cfg","modelCount":1,"models":{"mnist":{"1.0":{"defaultVersion":true,"marName":"mnist.mar","minWorkers":1,"maxWorkers":5,"batchSize":1,"responseTimeout":120}}}}
```

Comment:
took this from documentation https://kserve.github.io/website/0.8/modelserving/v1beta1/torchserve/#create-the-torchserve-inferenceservice

Reply:
also ran /v1/models and "mnist" is the model we print out